sourced/ml/core/algorithms/token_parser.py

```diff
@@ -76,6 +87,15 @@ def max_token_length(self, value):
             raise ValueError("max_token_length must be greater than 0 - got %d" % value)
         self._max_token_length = value
 
+    @property
+    def nn_model(self):
+        return self._nn_model
+
+    def _init_nn(self):
+        from sourced.ml.core.models.id_splitter import IdentifierSplitterBiLSTM
+        self._id_splitter_nn = IdentifierSplitterBiLSTM().load(
```

Comment:
```suggestion
        self._id_splitter_nn = IdentifierSplitterBiLSTM().load(self._nn_model)
```
still under 99 chars